Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -303,6 +304,7 @@ define([
         // actually generate commands.
         this.batchTable.update(tileset, frameState);
         this._model.modelMatrix = this._tile.computedTransform;
+        this._model.shadows = this._tileset.shadows;
```

Comment:
This looks good, except while testing I noticed that changing the shadows at run time wasn't working because the `Batched3DModel3DTileContent` clones the model's commands, so we need to also set the shadows for the cloned commands.

At https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Cesium3DTileBatchTable.js#L785 add the following lines:

```javascript
derivedCommands.originalCommand.castShadows = command.castShadows;
derivedCommands.originalCommand.receiveShadows = command.receiveShadows;
derivedCommands.back.castShadows = command.castShadows;
derivedCommands.back.receiveShadows = command.receiveShadows;
derivedCommands.front.castShadows = command.castShadows;
derivedCommands.front.receiveShadows = command.receiveShadows;
```